Formula/qpid-proton.rb

```diff
@@ -0,0 +1,64 @@
+class QpidProton < Formula
+  desc "is a high-performance, lightweight AMQP 1.0 messaging library."
+  homepage "https://qpid.apache.org/proton/"
+  url "https://www-eu.apache.org/dist/qpid/proton/0.17.0/qpid-proton-0.17.0.tar.gz"
+  sha256 "6ffd26d3d0e495bfdb5d9fefc5349954e6105ea18cc4bb191161d27742c5a01a"
+
+  depends_on "cmake" => :build
+  depends_on "libuv"
+  depends_on "openssl"
+  depends_on "perl" if Formula["perl"].installed?
```

Comment:
@MikeMcQuaid – I'd prefer to not depend on perl and python. The thing is, this library works very well with the system perl and python. But IF the homebrew packages are installed, I need to reference them for the compilation to work correctly. It would be a nuisance to force people to install the perl and python packages for no good reason. Some people prefer to work with the system packages (for example since they want to develop software that should run on plain-vanilla macOs), so we should give them a choice.